File: PSET6/cash.py
from cs50 import get_float
change = 0
quarter = 25
dime = 10
nickel = 5
quart_count = 0
dime_count = 0 
nickel_count = 0
total_count = 0
quart_mod = 0
dime_mod = 0
nickel_mod = 0
change = get_float("Change? ")
while change < 0:
    change = get_float("Change? ")
output = round(change * 100)
# Change output to quarters   
quart_count = output / quarter  
quart_mod = output % quarter     
# Print the output in terms of quarters   
print(f"In quarters:{int(quart_count)}")
# Print the output including dimes
if quart_mod != 0:
    dime_count = quart_mod / dime
    dime_mod = quart_mod % dime    
    print(f"In dimes of:{int(dime_count)}")  
# Print the output including nickels
if dime_mod != 0:
    nickel_count = dime_mod / nickel
    nickel_mod = dime_mod % nickel
    print(f"In nickel of:{int(nickel_count)}")           
# If there the nickels are not enough, add pennies
if nickel_mod != 0:
    print(f"In pennies of:{int(nickel_mod)}")   
# Getting the total count of all the coins in terms     
total_count = int(quart_count) + int(dime_count) + int(nickel_count) + int(nickel_mod)       
print(f"{total_count}")

File: PSET1/Cash/cash.c
#include<cs50.h>
#include<stdio.h>
#include<math.h>


int main(void)
{
    float change;
    int quarter = 25;
    int dime = 10;
    int nickel = 5;
    
    int quart_count = 0, dime_count = 0, nickel_count = 0, total_count = 0;
    int quart_mod = 0, dime_mod = 0, nickel_mod = 0;
    
    do
    {
        change = get_float("What's the change?");
    }
    
    
    while (change < 0);
    
    
    
    int output = round(change * 100);
    
    //Change output to quarters
    
    quart_count = output / quarter;
    
    quart_mod = output % quarter;
    
    
    //Print the output in terms of quarters
    
    printf("In quarters: %i \n", quart_count);
    
    //If there the quarters are not enough, add dimes
        
    if (quart_mod != 0)
    {
        
        dime_count = quart_mod / dime;
        
        dime_mod = quart_mod % dime;
        
        printf("In dimes of: %i \n", dime_count);
        
    }
    //If there the dimes are not enough, add nickels
        
    if (dime_mod != 0)
    {
        nickel_count = dime_mod / nickel;
        
        nickel_mod = dime_mod % nickel;
        
        printf("In nickel of: %i \n", nickel_count);
    }
    
    //If there the nickels are not enough, add pennies
    
    if (nickel_mod != 0)
    {
        
        printf("In pennies of: %i \n", nickel_mod);
    }
    
    
    
    //Getting the total count of all the coins in terms of quarters, dimes, nickels and pennies
     
    total_count = (quart_count + dime_count + nickel_count + nickel_mod);
        
    printf("%i \n", total_count);
    
    
    
    
    
}